commands/purge_config_command.go

```diff
@@ -52,11 +52,7 @@ func (c *PurgeConfigCommand) Execute(args []string) ExecutionStatus {
 		terminal.EntityNameColor(context.Space),
 		terminal.EntityNameColor(context.Username))
 
-	rc, err := c.NewRestClient(host)
-	if err != nil {
-		c.reportError(baseclient.NewClientError(err))
-		return Failure
-	}
+	rc := c.NewRestClient(host)
```

Comment:
Are you sure that it is not possible to return error while creating client?

Reply:
Yes.